src/modules/Users.pm

```diff
@@ -4476,149 +4307,24 @@ sub Write {
-		    {
-			$command = sprintf ("%s %s", $useradd_cmd, $username);
-			push @useradd_postcommands, $command;
-		    }
-		}
-		if ($user_mod eq "edited") {
-		    my $org_username	= $user{"org_user"}{"uid"} || $username;
-		    if ($username ne $org_username) {
-			y2usernote ("User '$org_username' renamed to '$username'");
-		    }
-		    else {
-			y2usernote ("User '$username' modified");
-		    }
-		}
-		if ($user_mod eq "edited" && $home ne "/var/lib/nobody") {
-		    my $org_home = $user{"org_user"}{"homeDirectory"} || $home;
-		    my $org_uid = $uid;
-		    $org_uid	= $user{"org_user"}{"uidNumber"} if (defined $user{"org_user"}{"uidNumber"});
-		    my $org_gid = $gid;
-		    if (defined $user{"org_user"}{"gidNumber"}) {
-			$org_gid = $user{"org_user"}{"gidNumber"};
-		    }
-		    # this would be actually caused by group modification
-		    elsif (defined $user{"org_gidNumber"}) {
-			$org_gid = $user{"org_gidNumber"};
-		    }
-		    # chown only when directory was changed (#39417)
-		    if ($home ne $org_home || $uid ne $org_uid || $gid ne $org_gid) {
-			# move the home directory
-			if (bool ($create_home)) {
-			    UsersRoutines->MoveHome ($org_home, $home);
-			}
-			# create new home directory
-			elsif (not %{SCR->Read (".target.stat", $home)}) {
-			    UsersRoutines->CreateHome ($skel, $home, "", $copy_usr_skel);
-			}
-			# do not change root's ownership of home directories
-			if (bool ($chown_home))
-			{
-			    UsersRoutines->ChownHome ($uid, $gid, $home);
-			}
-		    }
-		}
-
-		# Write authorized keys to user's home (FATE#319471)
-		if (defined($user{"authorized_keys"})) {
-		    SSHAuthorizedKeys->write_keys($home, $user{"authorized_keys"});
-		}
-
-                my %saved_passwords;
-
-                # hide passwords for logging
-                for my $pw (qw (userPassword text_userpassword)) {
-                    $saved_passwords{$pw} = $user{$pw};
-                    $user{$pw} = "*****";
-                }
-
-		y2milestone ("User = ", Dumper(\%user));
-
-		# restore
-		for my $pw (keys %saved_passwords) {
-		    $user{$pw} = $saved_passwords{$pw};
-		}
-	    }
-	}
-    }
-
-    WriteAuthorizedKeys();
-
-    # Write passwords
-    if ($use_gui) { Progress->NextStage (); }
-
-    if ($users_modified) {
-	if ($shadow_not_read) {
-	    # error popup (%s is a file name)
-            $ret = sprintf (__("File %s was not correctly read, so it will not be written."), $base_directory."/shadow");
-	    Report->Error ($ret);
-	    return $ret;
- 	}
-        if (! WriteShadow ()) {
-            $ret = Message->ErrorWritingFile ("$base_directory/shadow");
-	    Report->Error ($ret);
-	    return $ret;
-        }
+	# There used to be a big loop here managing homedir changes, but is not longer necessary:
+	#  - work with homes for LDAP users are ruled in WriteLDAP
+	#  - homes for local and system users are handed by Y2Users
+	#  - is not possible to add NIS users or to configure their homes
```

Comment:
these lines also causing troubles https://github.com/yast/yast-users/pull/333/files#diff-cf4196c4e5f27891c3cbd73e1ae939d14daea245d725677c7b51cdef7b833ad9R4367
it causes that org_user is removed and causes troubles to users reader